src/dcs-hook/OpenKneeboardDCSExt.lua

```diff
@@ -83,6 +102,15 @@ function callbacks.onSimulationStart()
   OpenKneeboard.send("SimulationStart", "");
 end
 
+function callbacks.onPlayerChangeSlot(id)
+	if id == net.get_my_player_id() then
+		local slotid = net.get_player_info(id,'slot')
+    state.aircraft = DCS.getUnitProperty(slotid, DCS.UNIT_TYPE)
+    l("Aircraft (onPlayerChangeSlot): "..state.aircraft)
+    OpenKneeboard.send("PlayerChangeSlot", state.aircraft);
```

Comment:
Given all that's being sent is the aircraft, perhaps EVT_AIRCRAFT should be re-used?

Consistency nit: 2 spaces instaed of 4 for indents